test/specs/format_spec.txt

```diff
@@ -5,11 +5,15 @@ ${'Asset {0}' @ format=properties.assetName}
 ${'Asset {0}' @ format=[properties.assetName]}
 ${'Asset {0} out of {1}' @ format=[properties.current, properties.total]}
 ${properties.formatter @ format=[properties.current, properties.total]}
+${properties.date @ format=properties.dateMilliseconds}
+${properties.date @ format=properties.assetName}
```

Comment:
would be nice to add more examples, like in https://github.com/adobe/htl-spec/blob/master/SPECIFICATION.md#examples-1

Reply:
Done added more date formatting and number formatting examples